app/Global.scala

```diff
@@ -186,12 +186,12 @@ Samplecountry
   }
 
   def insertProject = {
-    ProjectDAO.findAll(GlobalAccessContext).flatMap {
+    ProjectSQLDAO.findAll(GlobalAccessContext).flatMap {
       projects =>
         if (projects.isEmpty) {
           UserService.defaultUser.flatMap { user =>
-            val project = Project("sampleProject", organizationTeam._id, user._id, 100, false, Some(5400000))
-            for {_ <- ProjectDAO.insert(project)(GlobalAccessContext)} yield ()
+            val project = ProjectSQL(ObjectId.generate, ObjectId.fromBsonId(organizationTeam._id), ObjectId.fromBsonId(user._id), "sampleProject", 100, false, Some(5400000))
```

Comment:
Is `ObjectId.fromBsonId(some._id)` the same as `ObjectId(some)`, which is used below?

Reply:
The first takes a BSONObjectId as a parameter, the second takes a string.
When our refactoring is done, `fromBsonId` should not be necessary anymore, but right now this is the glue code